misk/src/test/kotlin/misk/web/ActionScopedWebDispatchTest.kt

```diff
@@ -0,0 +1,73 @@
+package misk.web
+
+import com.google.inject.util.Modules
+import misk.MiskModule
+import misk.inject.KAbstractModule
+import misk.scope.ActionScoped
+import misk.scope.ActionScopedProvider
+import misk.scope.ActionScopedProviderModule
+import misk.testing.ActionTest
+import misk.testing.ActionTestModule
+import misk.testing.TestWebModule
+import misk.web.actions.WebAction
+import misk.web.jetty.JettyService
+import misk.web.mediatype.MediaTypes
+import okhttp3.OkHttpClient
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.security.Principal
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@ActionTest(startService = true)
+internal class ActionScopedWebDispatchTest {
+    @ActionTestModule
+    val module = Modules.combine(
+            MiskModule(),
+            WebModule(),
+            TestWebModule(),
+            TestModule())
+
+    private @Inject lateinit var jettyService: JettyService
+
+    @Test
+    fun exposesActionScopedInInterceptors() {
+        val httpClient = OkHttpClient()
+        val response = httpClient.newCall(okhttp3.Request.Builder()
+                .url(jettyService.serverUrl.newBuilder().encodedPath("/hello").build())
```

Comment:
`WebDispatchTest` has a `serverUrlBuilder()` function that does some of this. Maybe it makes sense for `TestWebModule` to bind a `JettyTestSupport` that could be injected here to do some of this? Granted, `jettyService.serverUrl.newBuilder()` is hardly onerous, but I can see more functionality being added to such a class.

Reply:
Yeah, been thinking the same - there's a bunch of boilerplate around okhttp clients in these tests as well, might be good to pull these into helper TestSupport help functions